### ISSUE TITLE
NTR: Type II iodothyronine deiodinase (DIO2) low in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9306]
New term request Type II iodothyronine deiodinase (DIO2) low in fibroblasts (elthat_130419100011):

**Comment by pnrobinson** [comment]
@MickeySegal

does this need to be specifically fibroblast? What is the disease context?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/609698 " 2 families with a specific thyroid phenotype associated with a reduction in type II iodothyronine deiodinase DIO2; activity...Fibroblasts from the 3 sibs showed decreased DIO2 enzymatic activity"

Fibroblasts are likely the easiest tissue to check but it seems unlikely that the defect is specific to fibroblasts.

**Comment by pnrobinson** [comment]
@MickeySegal 
looking back at this, there was an article from 2005 that described the enzyme assay, but this does not appear to be a current clinical test -- instead, it was done for the research project, so there does not seem to be any gain from creating an HPO term for it?

**Comment by MickeySegal** [comment]
There are 20 citations to papers about the enzyme at https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4979226/ so it doesn't seem quite that obscure.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000719
- label: Reduced tissue type II iodothyronine deiodinase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: 
- def: Concentration or activity of type II iodothyronine deiodinase below the lower limit of normal. The activity can be measured in tissues such as cultured fibroblasts.
- comment: Type II iodothyronine deiodinase is a selenoprotein that catalyzes the 5-prime deiodination of thyroxine (T4) to generate an active thyroid hormone, 3,3-prime,5-triiodothyronine (T3).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9306
- synonyms:
